### DISCUSSION TITLE
Reolink Duo 2 support?

### DISCUSSION THREAD

**Discussion by John-OConor** [xannor/ha_reolink_rest#33] [Q&A] [answered]
I'm in the process of testing new Reolink camera's for my home and have purchased 2 Reolink Duo 2 cams.  I was just wondering if you had plans to implement those in your integration?  They do get discovered by "device discovery" and once the "IP Device" integration is loaded I do see 4 sensors active, just no active video entities.  This isn't surprising given Reolink has made several enhancements to the version 2 devices.  

I'm just curious if you have plans to look into / support this new version?  I would really like to use your integration if possible so I appreciate any feedback.  Thanks!  John

**Comment by xannor** [comment, marked as answer]
In theory they should, but I am working out some issues with multi-channel devices. I actually have a 1st Gen duo I test with. The biggest issue, and one I am "Detecting" and using HA's issue reporter for, in development, is that a lot of the newer firmware defaults to an HTTPS only setup that locks the camera down to only the literal web browser interface and their client. You have to go into the settings and enable things like RTSP/RTMP and ONVIF. These are needed to make the cameras work with 3rd party software, but dont support SSL themselves.

The other big problem is for anything 4K+ they only support H256, however RTMP does not, and the RTSP to WebRTC proxies dont either. This severely limits those cameras to RTSP for their hi-res streams, which does not work well with all browsers.
